src/utils/hooks/useKubevirtTranslation.ts

```diff
@@ -10,4 +10,4 @@ export const useKubevirtTranslation = () => useTranslation('plugin__kubevirt-plu
  * @param value string to translate
  */
 // skipcq: JS-C1002
-export const t = (value: string) => getI18n().t(value, { ns: 'plugin__kubevirt-plugin' });
+export const t: TFunction<'plugin__kubevirt-plugin', undefined> = getI18n().t;
```

Comment:
why remove the namespace? what is the default name space?

Reply:
the namespace is declared in the type, I changed it so we can also send translations like these:
`"{{cpu}} CPUS", { cpu: some_value }"`